aimsun_scripts/README.md

```diff
@@ -22,10 +22,10 @@ To create and execute the microsimulation in Aimsun, you must have a Aimsun base
 
 Once the Aimsun base network file and input data is available, please follow the following steps:
 
-1. Change `__REPO_PATH` in `append_github_path_to_python.py` to your absolute path for this repository (traffic-microsimulation).
+1. Change `__REPO_PATH` in `append_github_path_to_python.py` to your absolute path for this repository (traffic-microsimulation). Escape any special characters if necessary.
 2. Change `MICRO_BASELINE = True` in `create_simulations.py` and `run_simulation.py`.
 3. Open Aimsun base network file in Aimsun.
-4. Import all Python files in Aimsun by clicking File > Import > Python Script.
+4. Import all Python files in `aimsun_scripts`, `calibration`, and `utils` into Aimsun by clicking File > Import > Python Script.

```

Comment:
The user only needs to import files in the aimsun_scripts folder within Aimsun.
Could you modify this line to only include aimsun_scripts?
Thanks.

Reply:
Got it.